packages/calcite-components/.stylelintrc.cjs

```diff
@@ -1,7 +1,13 @@
 // @ts-check
 
 // ⚠️ AUTO-GENERATED CODE - DO NOT EDIT
-const customFunctions = [];
+const customFunctions = [
+  "get-trailing-text-input-padding",
+  "medium-modular-scale",
+  "modular-scale",
+  "scale-duration",
+  "small-modular-scale"
```

Comment:
Does this need a trailing comma to match the rest of the file? When this auto-generated for me it had one 🤔